<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleListWidget</class>
 <widget class="QWidget" name="RoleListWidget">
  <property name="windowTitle">
   <string>Role List</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="roleComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>User Roles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fullnameLabel">
       <property name="text">
        <string>fullname</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
